pilot/pkg/features/pilot.go

```diff
@@ -370,7 +370,7 @@ var (
 	XdsPushSendTimeout = env.RegisterDurationVar(
 		"PILOT_XDS_SEND_TIMEOUT",
 		5*time.Second,
-		"The timeout to send the XDS configuration to proxies. After this timeout is reached, Pilot will discard that push.",
+		"The timeout in seconds to send the XDS configuration to proxies. After this timeout is reached, Pilot will discard that push.",
```

Comment:
not sure this is accurate? you can put `PILOT_XDS_SEND_TIMEOUT=100ms` if you want

Reply:
Ah.. I missed that this is duration var :-) .Thanks